vm/errors.go

```diff
@@ -185,8 +185,8 @@ var ErrNilChanceComputer = errors.New("nil chance computer")
 // ErrNilEpochNotifier signals that the provided EpochNotifier is nil
 var ErrNilEpochNotifier = errors.New("nil EpochNotifier")
 
-// ErrNoTokenWithGivenName signals that token does not exists with given name
-var ErrNoTokenWithGivenName = errors.New("no token with given name")
+// ErrNoTickerWithGivenName signals that token does not exists with given name
```

Comment:
"ticker does not exist" instead "token does not exists"

Reply:
done